sdk/messaging/eventgrid/aznamespaces/models.go

```diff
@@ -25,15 +25,6 @@ type BrokerProperties struct {
 	LockToken *string
 }
 
-// Error - The error object.
-type Error struct {
```

Comment:
This is a bug in tcgc as the `Error` type is missing from the array of models. I opened https://github.com/Azure/typespec-azure/issues/1412 to track it.

Reply:
I'll add a comment in for a reminder. The nice thing is once it's fixed it'll be a compiler error for me :)